include/ttmlir/Target/Utils/MLIRToFlatbuffer.h

```diff
@@ -15,6 +16,30 @@
 #include "ttmlir/Utils.h"
 
 namespace mlir::tt {
+struct GoldenTensor {
+  std::string name;
+  std::vector<int64_t> shape;
+  std::vector<int64_t> strides;
+  ::tt::target::DataType dtype;
+  std::uint8_t *data;
```

Comment:
it's uint8_t because in Passes.cpp, I get the torch tensor data_ptr() (which is std::uintptr_t ptr) and I reinterpret cast it to unit8_t. It makes it easier to support the mismatch in ptr types